register-service/src/main/resources/sql/20211118_06_insert_mock_data_in_fitness_class_table.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO fitness_class (fitness_class_id, fitness_class_name)
+VALUES (1, 'yoga'),
+       (2, 'power lifting'),
+       (3, 'suria namaskar');
```

Comment:
please add constraints on slot table

Reply:
will be added later in scope of https://github.com/javaGuruBY/jis8-project/issues/44